Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Frostling.xml

```diff
@@ -20,13 +20,13 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_Frostling"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>0.15</ArmorRating_Blunt>
+					<ArmorRating_Blunt>2</ArmorRating_Blunt>

```

Comment:
Same as my comments on the Nightbeast; it's essentially a weird warg or wolf. Doesn't need much armor--wouldn't change it.